testinfra/modules/service.py

```diff
@@ -173,7 +173,14 @@ def is_running(self):
 
     @property
     def is_enabled(self):
-        raise NotImplementedError
+        # rcctl ls is only available on 5.8 or newer
+        if self.check_output('uname -r') >= '5.8':
+            for service in self.check_output("rcctl ls on").splitlines():
+                if service and service == self.name:
+                    return True
+            return False
+        else:
+            return NotImplementedError
```

Comment:
Ah! I didn't even think about what if the service doesn't even exist.

I'll push a new commit soon. Thanks for the feedback!